google/cloud/ndb/global_cache.py

```diff
@@ -559,7 +579,13 @@ def set(self, items, expires=None):
         """Implements :meth:`GlobalCache.set`."""
         items = {self._key(key): value for key, value in items.items()}
         expires = expires if expires else 0
-        self.client.set_many(items, expire=expires)
+        unset_keys = self.client.set_many(items, expire=expires, noreply=False)
+        if unset_keys:
+            warnings.warn(
+                "Keys failed to set in memcache: {}".format(unset_keys),
+                RuntimeWarning,
+            )
+            return {key: MemcacheCache.KeyNotSet(key) for key in unset_keys}
```

Comment:
shouldn't this be using the key passed into the method and not the cache_key?

Reply:
Yes, I had just caught that about ten minutes ago!